pkgs/by-name/gg/ggwave/package.nix

```
@@ -0,0 +1,36 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ggwave";
+  version = "unstable-2025-03-21";
```

[version must start with a digit](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)

```suggestion
  version = "1.5.2-unstable-2025-03-21";
```